demo/src/screens/componentScreens/TextFieldScreen.tsx

```diff
@@ -193,15 +194,17 @@ export default class TextFieldScreen extends Component {
           // validateOnStart
           // validateOnBlur
         />
-        <View row spread center>
+        <View row spread centerV>
           <TextField
             ref={this.inputWithValidation}
             label="Name"
             placeholder="Enter full name"
             validate="required"
-            validationMessage="This field is required"
-            containerStyle={{flexGrow: 1}}
+            validationMessage="This field is required. That means you have to enter some value"
+            containerStyle={{width: 300}}
```

Comment:
It's not a good fix, we can't expect the user to do this, so we need to understand better the issue maybe it'll lead us to a problem in the TextField implmentation
We can do it separately since it's not related to the PR as you said

Reply:
fixed